docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc

```diff
@@ -12,13 +12,9 @@ You have a Hibernate ORM-based application? You want to provide a full-featured
 With this guide, you'll learn how to synchronize your entities to an Elasticsearch cluster in a heart beat with Hibernate Search.
 We will also explore how you can can query your Elasticsearch cluster using the Hibernate Search API.
 
-[WARNING]
```

Comment:
Maybe let's keep it WARNING for the ones with beta components (so the two different ones).

Reply:
Done.